test_integration.py

```diff
@@ -110,10 +110,13 @@
     x = s_int.get_error()
     x[:] /= 2
     fl.next('error bars')
-    fl.plot(s_int.get_error(),'o',label='returned error from integral_w_errors')
-    avg_s_int = s_int.get_error().mean().item()
+    int_error = s_int.get_error()
+    fl.plot(int_error,'o',
+            label='returned error from integral_w_errors')
+    avg_s_int = int_error.mean().item()
```

Comment:
this is just formatting.

this reminds me -- in the future, we will use [black](https://github.com/psf/black) We don't want to do that as part of this pull request, though, since it will make a million small changes to the file.